.github/workflows/main.yml

```
@@ -211,3 +163,9 @@ jobs:
           github_token: ${{ secrets.github_token }}
           reporter: github-pr-review
           level: error
+
+  test-sdk-builds:
+    name: Test SDK builds
+    uses: ./.github/workflows/generate-and-build-sdks.yml
+    with:
+      xapi_version: "0.0.0"
```

is there anything against
```suggestion
      xapi_version: ${{ env.XAPI_VERSION }}
```

---

honestly just the fact that I don't want people thinking that the PR artefacts are valid for real usage. 

I know we also flag release ones as unsigned and prerelease but these are generated for every PR and I wanted to make it even clearer that they should not be used at all

---

XAPI_VERSION should be 0.0.0 most of the time. I'm just wary of having 2 sources of truth

---

ah! that's a good point - I'll change it 👍

---

done, see https://github.com/xapi-project/xen-api/compare/15e55edc3e51bfc8d3b9a88ea61b2da32052dba8..361e342b9c6668643c28c09bcb14571af0aa6540